spring-cloud-stream/pom.xml

```diff
@@ -50,6 +50,7 @@
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-stream-binder-redis</artifactId>
+			<optional>true</optional>
```

Comment:
If we do this (which is sane), we need to change our examples - the build currently fails building them.

https://travis-ci.org/spring-cloud/spring-cloud-stream/builds/74914672